### ISSUE TITLE
GameGuru MAX - Bug - Changing Texture Not Saved

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#3536]
If I select an object, go to the Materials tab, and select a new Albedo texture for it, after saving the level and restarting MAX, the original texture is still applied and the new one forgotten. As a result, I cannot change a texture on an individual object if I want to.

**Comment by Kitakazi** [comment]
I've been having something similar happening since the last update. 

I select an object, go to Materials tab and select a new Albedo Texture, or I change the color of the Albedo texture, once I select a different entity it will change back to its previous state before I made any changes. Sometimes simply selecting this entity again will switch it back to the changes I had just made. 

Seems this bug has plagued GGMax since its creation.
